packages/frontend-2/components/settings/workspaces/General.vue

```diff
@@ -224,4 +258,50 @@ onResult((res) => {
     defaultProjectRole.value = res.data.workspace.defaultProjectRole as StreamRoles
   }
 })
+
+const config = useRuntimeConfig()
+const baseUrl = config.public.baseUrl
+
+const slugHelp = computed(() => {
+  return `Used after ${baseUrl}/workspaces/`
```

Comment:
Agree that this seems weird, but I spoke to Michal and there is no great alternative. He wants to keep this url preview messaging, but the preface to the url needs to be short to avoid it dropping onto 2 lines. 
<img width="630" alt="image" src="https://github.com/user-attachments/assets/c24d3ae7-701b-423e-8908-ece4e0e1b308">